### DISCUSSION TITLE
No new features to 1.x

### DISCUSSION THREAD

**Discussion by saeedvaziry** [vitodeploy/vito#308] [Announcements]
As already started working on v2, we're not merging features other than Bugs to the v1

**Comment by Noaber** [comment]
Is v2 recommended for new server or should we use v1 and later upgrade (is possible) to v2 ?

**Reply by saeedvaziry** [reply]
v1 is the stable release at the moment.

**Comment by kperrow** [comment]
Hi @saeedvaziry,
Will version 1.x be updatable to V2 when its released or is it going to be a whole new build?

**Reply by saeedvaziry** [reply]
of course, it will be upgradable. the upgrade guide gonna be super simple
